src/binary/uint.rs

```diff
@@ -40,16 +40,16 @@ impl UInt {
             magnitude |= byte;
         }
 
-        Ok(UInt {
+        Ok(DecodedUInt {
             size_in_bytes: length,
             value: magnitude,
         })
```

Comment:
Not your change, but I really think we need to note why this type is what it is, since this length might be padded or not directly relevant to the decoded value...